fasthtml/js.py

```diff
@@ -87,3 +88,47 @@ def SortableJS(
 proc_htmx('%s', Sortable.create);
 """ % sel
     return Script(src, type='module')
+
+# %% ../nbs/api/03_js.ipynb
+def MermaidJS(
+        sel='.language-mermaid',  # CSS selector for mermaid elements
+        theme='base',  # Mermaid theme to use
+        delay=500  # Delay in milliseconds before rendering
+    ):
+    "Implements browser-based Mermaid diagram rendering."
+    src = """
```

Comment:
This should be in a stand-alone JS file rather than a Python string template called with something like Script(rel='/mermaid.js').

Reply:
If you have a look at our other examples Danny we normally use ScriptX or python strings. Otherwise you have to deal with more complex file inclusion during deployment

Reply:
Apologies, I thought we had moved to a JS-file approach. Based on your response I just looked at https://github.com/AnswerDotAI/fasthtml-js and it wasn't what I thought it was.